.npmrc

```diff
@@ -1 +1,3 @@
 git-tag-version=false
+prefer-dedupe=true
+save-exact=true
```

Comment:
With the new dependency system, we use the pinned dependencies, so we can easily remove and re-create `package-lock.json` without pulling dependency inconsistence. To update dependencies, we can use [`npm-check-updates`](https://www.npmjs.com/package/npm-check-updates) package